tests/util/tests_CowString.cpp

```diff
@@ -1,13 +1,13 @@
 #define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 
 #include <doctest/doctest.h>
-#include <rdf4cpp/rdf.hpp>
+#include <rdf4cpp.hpp>
 
-using namespace rdf4cpp::rdf::util;
+using rdf4cpp::CowString;
```

Comment:
nothing we should tackle in this PR but please create an issue to move CowString to dice-template-library

Reply:
https://github.com/dice-group/dice-template-library/issues/38